Backpack-SwiftUI/Tests/Font/FontTests.swift

```diff
@@ -0,0 +1,42 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+import XCTest
+@testable import Backpack_SwiftUI
+
+class FontTests: XCTestCase {
```

Comment:
re tests: i was thinking of the screenshot tests we have for uikit with all the different font styles. but not sure if it's done as part of the typography styles or `BpkText`

Reply:
Those are not needed now, because the font styles are accessible at compiletime and dont need to be 'calculated' via multiple enum cases and some other magic for them to work!

Testing that they exist now would just be checking if they are there at compile time which would not be testing behaviour of the font Token, plus it will just add complexity whenever we decide to add/modify/remove font tokens.